pages/_app.tsx

```diff
@@ -0,0 +1,30 @@
+import { AppProps } from 'next/app';
+import { RecoilRoot } from 'recoil';
+import Head from 'next/head';
+import { ColorScheme, ColorSchemeProvider, MantineProvider } from '@mantine/core';
+import { useState } from 'react';
+import { NotificationsProvider } from '@mantine/notifications';
+
+export default function App(props: AppProps) {
+  const { Component, pageProps } = props;
+  const [colorScheme, setColorScheme] = useState<ColorScheme>('light');
+  const toggleColorScheme = (value?: ColorScheme) =>
+    setColorScheme(value || (colorScheme === 'dark' ? 'light' : 'dark'));
+
+  return (
+    <RecoilRoot>
+      <Head>
+        <title>FQM Testify: an ECQM Analysis Tool</title>
+        <meta name="viewport" content="minimum-scale=1, initial-scale=1, width=device-width" />
+        <link rel="icon" href="/favicon.ico" />
```

Comment:
Do you know what this header does or where it appears? I tried deleting it and didn't notice any changes. Do we need this?

Reply:
Cool. Thanks for the research!